homework/01LessonOne.js

```diff
@@ -204,11 +207,11 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // Your Answer Goes Here:
 //
-//
+// == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type '8'=== 8 is false because they are not the same value. '8' is a string not a variable and 8 would be a variable.
 // What’s the difference between != and !==?
```

Comment:
Yes!!  I just wanted to make sure me thinking that was correct.  Thank you for your help! 

Sent from my iPhone

> On Jul 31, 2016, at 3:34 PM, Piero A. Pretto <notifications@github.com> wrote:
> 
> In homework/01LessonOne.js:
> 
> > @@ -204,11 +207,11 @@ var stringFourStrictNotEqualsNumberFour;
> >  //
> >  // Your Answer Goes Here:
> >  //
> > -//
> > +// == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type '8'=== 8 is false because they are not the same value. '8' is a string not a variable and 8 would be a variable.
> >  // What’s the difference between != and !==?
> I understand. You answered the corresponding questions correctly; just wanted to make sure the concept was crystal clear.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>